roles/openshift_health_checker/library/etcdlogs.py

```diff
@@ -0,0 +1,95 @@
+# pylint: disable=missing-docstring
+
+"""Interface to journalctl"""
```

Comment:
I would just have it pass back what matched which regex and let the caller figure out how to respond.